libyara/modules/math/math.c

```diff
@@ -738,6 +787,10 @@ begin_declarations
   declare_function("percentage", "i", "f", percentage_global);
   declare_function("mode", "ii", "i", mode_range);
   declare_function("mode", "", "i", mode_global);
+  declare_function("inttostring", "i", "s", inttostring);
```

Comment:
What about naming the functions `math.to_int` and `math.to_str`? If not, I would follow the snake case convention `int_to_string` and `string_to_int`, or (`int_to_str` and `str_to_int` in abbreviated form)